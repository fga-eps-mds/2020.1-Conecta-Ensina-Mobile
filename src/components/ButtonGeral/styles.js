import styled from 'styled-components';

export const ButtonGeral = styled.TouchableOpacity`
  background-color: ${(props) => props.theme.colors.fundoAzul};
  height: 50px;
  width: 200px;
  border-radius: 10px;
`;
