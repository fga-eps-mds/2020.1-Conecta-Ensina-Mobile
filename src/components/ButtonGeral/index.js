import React from 'react';

import {ButtonGeral} from './styles';

export default function BotaoGeral() {
  return <ButtonGeral/>;
}
