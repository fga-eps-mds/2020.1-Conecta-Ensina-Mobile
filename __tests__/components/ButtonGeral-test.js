import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';

import ButtonGeral from '../../src/components/ButtonGeral';

it('renders correctly', () => {
  const tree = renderer.create(<ButtonGeral />).toJSON();
  expect(tree).toMatchSnapshot();
});
