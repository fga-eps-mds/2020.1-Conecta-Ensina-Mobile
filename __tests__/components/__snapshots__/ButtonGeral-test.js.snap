// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#184E79",
      "borderRadius": 10,
      "height": 50,
      "opacity": 1,
      "width": 200,
    }
  }
>
  <Text
    small={true}
    style={
      Array [
        Object {
          "color": "#ffffff",
          "fontFamily": "Roboto",
          "fontSize": 14,
          "textAlign": "center",
        },
      ]
    }
    white={true}
  />
</View>
`;
